src/plugins/convert-chinese.ts

```diff
@@ -31,10 +31,14 @@ export const convertChinese = async (directories: string) => {
       console.info(`Converting ${file}...`);
       const fileText = await readFile(file, "utf-8");
       const translatedText = await opencc.simplifiedToTraditional(fileText);
-      await outputFile(
-        file.replace("chinese", "chinese-traditional"),
-        translatedText
-      );
+      if (process.env.USE_LANG_CODE) {
+        await outputFile(file.replace("zh", "zh-TW"), translatedText);
+      } else {
+        await outputFile(
+          file.replace("chinese", "chinese-traditional"),
+          translatedText
+        );
+      }
```

Comment:
I am struggling to word this:

Seeing as I plan to set the Rust in Replit course up with 2_KEY_CODEs, how will Crowdin handle the next upload after a download creating a `zh-TW` folder?

Reply:
With the config we walked through, you tell Crowdin to upload only the `/en` folder. The upload process won't look at `zh` or `zh-TW`.

Reply:
@ShaunSHamilton in other words: 

English language versions from all source code repositories are the only things that need to be uploaded. 

We assume whatever happens on Crowdin to be the source of truth and download/overwrite the localized files to the code repositories. This is why we do not allow changes to localization on GitHub because they get overwritten.

Yes, the means sometimes when we do hard pivots (say structural changes) we also "manually" upload the localized content to Crowdin, but we need them approved by proofreaders.